src/components/utils/error/errorCode.js

```diff
@@ -13,3 +13,5 @@ export const ERROR_CODES = {
     ERR_LIMIT_REACHED: "ERR_LIMIT_REACHED",
     ERR_PERMISSION_NEEDED: "ERR_PERMISSION_NEEDED",
 };
+
+export const signInErrorResponse = { response: { status: 401 } };
```

Comment:
🎉 . @flynn1999 - This is what I was referring to in my comments in https://github.com/cyverse-de/sonora/pull/306#discussion_r607223196